Nasal/electrical.nas

```diff
@@ -79,7 +84,7 @@ BatteryClass.new = func {
 #
 
 BatteryClass.apply_load = func (amps, dt) {
-    var old_charge_percent = getprop("/systems/electrical/battery-charge-percent");
+    var old_charge_percent = me.charge_percent;
```

Comment:
@onox my bad again :disappointed: This line didn't need to be changed, the line that makes the trick is the new line 99 below (`me.charge_percent = new_charge_percent;`). So should I revert this line and remove those spaces added below?